src/crx.js

```diff
@@ -6,6 +6,7 @@ var crypto = require("crypto");
 var RSA = require("node-rsa");
 var archiver = require("archiver");
 var resolve = require("./resolver.js");
+var crx3 = require("./crx3.js");
```

Comment:
Not necessary for this PR, but it'd be nice if the CRX2 implementation was pulled out into its own file too.

Reply:
@arkon Yes, definitely. I mentioned it in this PR - i just did not want to introduce too many changes in a single PR. Especially that when i worked on it, CRXv3 was not mandatory in Chrome yet.